static/mappings/headerMenu.properties

```diff
@@ -1,15 +1,12 @@
-menu.order=home,browse,about,contact
+menu.order=home,collections,departments
 menu.home.label=Home
 menu.home.url=/
-menu.browse.label=Browse
-menu.browse.url=/collections
-menu.browse.order=collections,depts
-menu.browse.entry.collections.label=Browse Collections
-menu.browse.entry.collections.url=/collections
-menu.browse.entry.depts.label=Browse Departments
-menu.browse.entry.depts.url=/browse/dept/
-menu.about.label=About
-menu.about.url=http://blogs.lib.unc.edu/cdr/
-menu.contact.label=Contact
-menu.contact.url=http://blogs.lib.unc.edu/cdr/index.php/contact-us/
-menu.contact.referer=true
+menu.collections.label=Collections
```

Comment:
Have the feeling we're getting to the point where we should just hardcode the menues in html since we're down to two buttons, but that doesn't need to be part of this PR